packages/google_sign_in/android/build.gradle

```diff
@@ -27,13 +27,14 @@ android {
     defaultConfig {
         minSdkVersion 16
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles 'lib-proguard-rules.txt'
     }
     lintOptions {
         disable 'InvalidPackage'
     }
 }
 
 dependencies {
-    api 'com.google.android.gms:play-services-auth:15.+'
-    implementation 'com.google.guava:guava:20.0'
+    api 'com.google.android.gms:play-services-auth:16.0.0'
+    api 'com.google.guava:guava:26.0-android'
```

Comment:
If we just remove this, the plugin should still work as far as I can tell.  Can you test that?  If so, we wouldn't need the special proguard rules.